sympy/core/function.py

```diff
@@ -2071,9 +2087,20 @@ class Subs(Expr):
     """
     Represents unevaluated substitutions of an expression.
 
-    ``Subs(expr, x, x0)`` receives 3 arguments: an expression, a variable or
-    list of distinct variables and a point or list of evaluation points
-    corresponding to those variables.
+    ``Subs(expr, x, x0)`` receives 3 arguments.
+
+    Paramters
+    =========
```

Comment:
Parameters

Reply:
Changed.

Reply:
This still have the matching underbar length problem.
See https://travis-ci.org/github/sympy/sympy/jobs/731335745#L1370